tensorboard/plugins/hparams/tf_hparams_query_pane/tf-hparams-query-pane.html

```diff
@@ -765,6 +804,9 @@
       // above. Returns a promise, for use in unit-tests, that resolves (with
       // 'undefined') when the ListSessionGroups RPC returns.
       _queryServerNoDebounce() {
+        if (!this._hparams || !this._metrics) {
+          return null;
```

Comment:
nit: `return null` --> `return`, since nothing depends on the return value?

Reply:
Done